src/Serilog.Sinks.RollingFile/Sinks/RollingFile/TemplatedPathRoller.cs

```diff
@@ -29,18 +29,42 @@ class TemplatedPathRoller
         const string OldStyleDateSpecifier = "{0}";
         const string DateSpecifier = "{Date}";
         const string DateFormat = "yyyyMMdd";
+        const string HourSpecifier = "{Hour}";
+        const string HourFormat = "yyyyMMddHH";
+        const string HalfHourSpecifier = "{HalfHour}";
+        const string HalfHourFormat = "yyyyMMddHHmm";
         const string DefaultSeparator = "-";
 
+        const string MatcherMarkSpecifier = "date";
+        const string MatcherMarkInc = "inc";
+
         readonly string _pathTemplate;
         readonly Regex _filenameMatcher;
+        readonly SpecifierTypeEnum _specifierType = SpecifierTypeEnum.None;
```

Comment:
Carrying both `_specifierType` and `_usedSpecifier` means the two could (potentially) be out-of-sync. What do you think about getting rid of the enum, and just using e.g. `_usedSpecifier == HourSpecifier` etc. as comparisons?